daras_ai_v2/base.py

```diff
@@ -343,6 +345,12 @@ def render(self):
         self._user_disabled_check()
         self._check_if_flagged()
 
+        if self.should_submit_after_login():
```

Comment:
SAVE_AFTER_LOGIN_Q and SUBMIT_AFTER_LOGIN_Q shold be mutually exclusive

Reply:
done